dpnp/tests/test_random_state.py

```diff
@@ -153,20 +149,20 @@ def test_inf_loc(self, loc):
         )
 
     def test_inf_scale(self):
-        a = RandomState().normal(0, numpy.inf, size=1000).asnumpy()
-        assert_equal(numpy.isnan(a).any(), False)
-        assert_equal(numpy.isinf(a).all(), True)
+        a = RandomState().normal(0, numpy.inf, size=1000)
+        assert_equal(dpnp.isnan(a).any(), False)
```

Comment:
we can use the usual assert here:
```suggestion
        assert not dpnp.isnan(a).any()
```